ig/README.md

```diff
@@ -20,10 +20,11 @@ To be written
 1. run `./_updatePublisher.sh` in order to ensure you are using the most recent publisher release
 1. run `./_genonce.sh` to generate the Implementation Guide
 1. In a browser you can open up the `ig/output/index.html` file that was generated as this is the home page for the Implementation Guide
+2. Alternately, you may opt to use the local development Docker approach outlined below to avoid having to install the toolchain listed above.
```

Comment:
Mind making this number a `1` to match?  Either that, or make them all ascend.
```suggestion
1. Alternately, you may opt to use the local development Docker approach outlined below to avoid having to install the toolchain listed above.
```

Reply:
Ah, I'll fix this.  I believe the markdown validator wants this to be fixed as well